setup.py

```diff
@@ -25,7 +25,7 @@ def read_file(filename):
 
 setup(
     name="kinto-dist",
-    version="27.0.0",
```

Comment:
Should we read `VERSION` here too?

Reply:
I saw that too, I think we should just ditch the packaging of kinto-dist itself in another PR. Let me do that